elytron/src/test/resources/org/wildfly/extension/elytron/tls-ibm.xml

```diff
@@ -122,14 +122,15 @@
                                 session-timeout="321" maximum-session-cache-size="123"/>
             <server-ssl-context name="ServerSslContextSSLv2Hello" protocols="TLSv1 SSLv2Hello"
                                 key-manager="ServerKeyManager" trust-manager="CaTrustManager" want-client-auth="true" need-client-auth="true"
-                                authentication-optional="false" providers="ManagerProviderLoader" provider-name="IBMJSSE"
+                                authentication-optional="false" providers="ManagerProviderLoader"
```

Comment:
To make sure the IBM provider is actually used for this test, instead of removing the `provider-name`, we could update it to `IBMJSSE2`. That seems to be the approach used for the other ssl-contexts configured in this file.